wiki/Guides/How_to_Use_the_Offset_Wizard/en.md

```diff
@@ -1,3 +1,25 @@
 # How to use the offset wizard
+*See also: [Universal Offset](/wiki/Universal_offset)*
 
-This guide was incorrect and has been removed.
+If the [Hitsounds](/wiki/Beatmapping/Hitsound) feel out of sync with the music, in every beatmap you play, you will more than likely need to change your [Universal Offset](/wiki/Universal_offset). The easiest way to do this is by using the **Offset Wizard**.
+
+Follow these steps to adjust your Universal Offset using the Offset Wizard:
+
+**Opening the Offset Wizard**:
+
+1. Open osu!.
+2. Go to the `Options` menu, or press `Ctrl` + `O`.
+3. Search "offset" in the quick-search and click on `Offset wizard`.
+
+The Offset Wizard will then open, playing the song that was previously playing in osu!, while a green bar scrolls across the screen and a metronome plays.
+
+**Using the Offset Wizard to calibrate your Universal Offset**:
+
+1. Listen carefully to the song and the metronome playing together.
```

Comment:
```suggestion
1. Listen carefully to the song and the metronome ticking together.
```

Reply:
the song and the metronome dont "tick" together though, so i'm not sure this makes sense